xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -246,6 +246,7 @@ For more information, see [How to serialize and deserialize JSON](~/docs/standar
 
  ]]></format>
         </remarks>
+        <exception cref="NotSupportedException">There is no compatible <see cref="System.Text.Json.Serialization.JsonConverter"/> for <paramref name="typeToConvert"/> or its serializable members.</exception>
```

Comment:
What about the `InvalidOperationException`: https://github.com/dotnet/runtime/pull/33400/files#diff-bc4374ba33eacef0bf43a48fd7059eb9R117-R119

Reply:
Added. Thanks

Reply:
I wonder why this exception was not ported automatically. @steveharter do you see the exception in the xml generated after you built runtime?